scriptmodules/supplementary/runcommand/runcommand.sh

```diff
@@ -778,6 +784,13 @@ function choose_emulator() {
     local default="$2"
     local cancel="$3"
 
+    local mode_desc="default emulator for "
+    if [[ "$mode" == "emu_sys" ]]; then
+        mode_desc+="$SYSTEM"
+    else
+        mode_desc+="ROM ($ROM)"
+    fi
+
```

Comment:
Might want to remove `default` re-declaration and move `default_id` at the top of the function.

Reply:
Will check this. It's not really a redeclaration in bash but will clean up if needed.

Reply:
Ah thought it was something in the PR. Yeah I'll remove the second declaration. Not moving the default_id declaration as it's not c89 ;-)